test/widgets/action_sheet_test.dart

```diff
@@ -551,6 +554,21 @@ void main() {
       final expectedLink = narrowLink(store, narrow, nearMessageId: message.id).toString();
       check(await Clipboard.getData('text/plain')).isNotNull().text.equals(expectedLink);
     });
+
+    testWidgets('Full Url Testing', (tester) async {
+      final message = eg.streamMessage();
+      final narrow = TopicNarrow.ofMessage(message);
+      await setupToMessageActionSheet(tester, message: message, narrow: narrow);
+      await tapCopyMessageLinkButton(tester);
+      await tester.pump(Duration.zero);
```

Comment:
nit: Move this to just before the line where we define `expectedLink`, so the reader gets the contextual information about where the clipboard data comes from